src/styles/styles.js

```diff
@@ -1547,8 +1547,31 @@ const styles = {
             // paddingVertical: 0, alignSelf: 'center', and textAlignVertical: 'center'
 
             paddingHorizontal: variables.avatarChatSpacing,
-            paddingTop: 0,
-            paddingBottom: 0,
+            paddingTop: 9,
+            paddingBottom: 9,
+            alignSelf: 'center',
+            textAlignVertical: 'center',
+        },
+        0,
+    ),
+    textInputComposeMultiLines: addOutlineWidth(
+        {
+            backgroundColor: themeColors.componentBG,
+            borderColor: themeColors.border,
+            color: themeColors.text,
+            fontFamily: fontFamily.EMOJI_TEXT_FONT,
+            fontSize: variables.fontSizeNormal,
+            borderWidth: 0,
+            height: 'auto',
+            lineHeight: variables.lineHeightXLarge,
+            ...overflowXHidden,
+
+            // On Android, multiline TextInput with height: 'auto' will show extra padding unless they are configured with
+            // paddingVertical: 0, alignSelf: 'center', and textAlignVertical: 'center'
```

Comment:
I'm not sure what to make of this. Why is this explanation here, but it doesn't really appear to do anything or be valuable.